modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJuliaCodegen.java

```diff
@@ -531,6 +531,14 @@ private void changeParamNames(List<CodegenParameter> paramsList, HashSet<String>
         }
     }
 
+    private void changeRegexEscape(List<CodegenParameter> paramsList) {
```

Comment:
minor suggestion: what about adding a docstring explaining what this function does?

Reply:
sure, added in https://github.com/OpenAPITools/openapi-generator/pull/21009/commits/c445ea22cb7cf4b139a1201c7dbe4a20460fa044

Reply:
what about simply overriding the method `toRegularExpression` to fix the issue instead post processing all parameters?

Reply:
Thanks for the suggestion, that actually simplifies things. Done in https://github.com/OpenAPITools/openapi-generator/pull/21009/commits/3bc26f6af3cfda90bea18f8f67565e6225997f4e